src/bin/pg_autoctl/monitor_pg_init.c

```diff
@@ -32,6 +32,13 @@
  */
 GUC monitor_default_settings[] = {
 	{ "shared_preload_libraries", "'pgautofailover'" },
+	{ "log_destination", "stderr"},
```

Comment:
@DimCitus  we should re-evaluate if we want to have these changes.

Reply:
I think it would be best if we can isolate them in another PR...

Reply:
removed from this pr